secure_config.py

```
@@ -0,0 +1,214 @@
+        If encryption fails for any reason, logs a warning and returns plain text.
+    """
+    if not ENCRYPTION_AVAILABLE:
+        logger.warning("Encryption requested but cryptography not installed - returning plain text")
+        return value  # Fallback to plain text if crypto not available
+
+    try:
+        key = get_or_create_encryption_key()
+        fernet = Fernet(key)
+        encrypted = fernet.encrypt(value.encode())
+        return f"encrypted:{encrypted.decode()}"
+    except Exception as e:
```

Ruff (0.13.1)

Do not catch blind exception: `Exception`

(BLE001)